win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -588,20 +587,13 @@ TEST_FUNCTION(MU_C3(starting_, N_TIMERS, _timer_start_runs_once))
     // open
     ASSERT_ARE_EQUAL(int, 0, threadpool_open(threadpool));
 
-    TIMER_INSTANCE_HANDLE timers[N_TIMERS];
-    for (uint32_t index = 0; index < 1; index++)
-    {
-        ASSERT_ARE_EQUAL(int, 0, threadpool_timer_start(threadpool_1, 200, 200, work_function, (void*)&call_count, &timers[index]));
-
-    }
+    THANDLE(TIMER_INSTANCE) timer_instance = NULL;
+    ASSERT_ARE_EQUAL(int, 0, threadpool_timer_start(threadpool_1, 200, 200, work_function, (void*)&call_count, &timer_instance));
```

Comment:
>timer_instance [](http://example.com/codeflow?start=112&length=14)

We should assert that we got a non-NULL timer

Reply:
added